.github/workflows/build_main_documentation.yml

```
@@ -144,9 +146,16 @@ jobs:
         run: |
           cd doc-build
           git pull
-          mv ../build_dir/${{ inputs.package }}/_versions.yml ${{ inputs.package }}/
-          rm -rf ${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }})
-          mv ../build_dir/${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }}) ${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }})
+          if [ -z "${{ inputs.package_name }}" ];
+          then
+            mv ../build_dir/${{ inputs.package }}/_versions.yml ${{ inputs.package }}/
+            rm -rf ${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }})
+            mv ../build_dir/${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }}) ${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }})
+          else
+            mv ../build_dir/${{ inputs.package }}/_versions.yml ${{ inputs.package_name }}/
+            rm -rf ${{ inputs.package_name }}/$(ls ../build_dir/${{ inputs.package }})
+            mv ../build_dir/${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }}) ${{ inputs.package_name }}/$(ls ../build_dir/${{ inputs.package }})
```

Should the target folder also be `package_name`?

```suggestion
            mv ../build_dir/${{ inputs.package }}/$(ls ../build_dir/${{ inputs.package }}) ${{ inputs.package_name }}/$(ls ../build_dir/${{ inputs.package_name }})
```

---

in this cased, I think we still want `package` because of `$(ls ../build_dir/${{ inputs.package_name }})`